material/problems/Kinematics/SwingingShip/SwingingShip.tex

```diff
@@ -24,48 +24,110 @@
-  $\omega_B$. The distance from $B_o$ to the bottom surface of the ship is
-  $R_B = 6$ m.
+  $\omega_B$.
+  \\[0.45pc]
+  The radius of the tire is $R_A = 0.3$ m, and the distance from $B_o$ to the
+  bottom surface of the ship is $R_B = 6$ m.
   %
-  \\[0.5pc]
-  Right-handed orthogonal unit vectors \uvecxyz{n} are fixed in \basis{N}
-  as shown.
+  %\\[0.45pc]
+  %Right-handed orthogonal unit vectors \uvecxyz{n} are fixed in \basis{N}
+  %as shown.
   %
-%  \begin{center}
-%    \includegraphics[height=6cm]{mayflower.jpg}
-%    \hspace{1cm}
-%    \includegraphics[height=6cm]{boat.png}
-%  \end{center}
+  \begin{center}
+    \includegraphics[height=6cm]{mayflower.jpg}
+    \hspace{1cm}
+    \includegraphicsAB[height=6cm]{boat-solution.png}{boat.png}
+  \end{center}
   %
-  %\\[0.5pc]
-  \begin{itemize}
+  \begin{enumerate}
     \item \textbf{Calculate} the velocity of the point of the tire in contact
-      with the ship, $\vel{A_B}{N}$.
-      %
-    \item \textbf{Calculate} the velocity of the point of the ship in contact
-      with the tire, $\vel{B_A}{N}$.
+      with the ship, $\vel{A_B}{N}$, and calculate the velocity of the point of
+      the ship in contact with the tire, $\vel{B_A}{N}$.
       %
+    \\[0.45pc]
+    \Solution{}{0.99\linewidth}{
+      \begin{tabular}{@{}l@{ \equals[\;] }l}
+        $\vel{A_B}{N}$
+        & $\vel{A_o}{N}\plus{}^N\vec{\omega}^A \times\vec{r}^{A_B\slash A_o}$
+        \\[0.5pc]
+        &$\vec{0} \plus(\omega_A \hat{n_z}) \times(R_A \hat{n_y})$
+        \\[0.5pc]
+        &$\omega_AR_A(\minus\hat{n_x})$
+      \end{tabular}
+      \hspace{1cm}
+      \begin{tabular}{@{}l@{ \equals[\;] }l}
+        $\vel{B_A}{N}$
+        & $\vel{B_o}{N}\plus{}^N\vec{\omega}^B \times\vec{r}^{B_A \slash B_o}$
+        \\[0.5pc]
+        &$\vec{0} \plus (\omega_B \hat{n_z}) \times (\minus R_B \hat{n_y})$
+        \\[0.5pc]
+        &$\omega_B R_B(\hat{n_x})$
+      \end{tabular}
+      \\[2.0pc]
+    }
+	    %
     \item When the ship \textbf{rolls} on the tire, write the relevant
       \textbf{constraint equation} and use it to find a \textbf{scalar}
       relationship between $\omega_A$ and $\omega_B$.
       %
+    \\[0.45pc]
+    \Solution{}{0.99\linewidth}{
+      \begin{tabular}{l@{}r@{\equals[\;]}l}
+        Rolling Constraint:
+        &$\vel{A_B}{N}$
+        &$\vel{B_A}{N}$
+        \\[0.5pc]
+        &$\minus \omega_A R_A \hat{n_x}$
+        &$\omega_B R_B \hat{n_x}$
+        \\[0.5pc]
+        dot with $\hat{n_x}$:
+        &$\minus \omega_A R_A$
+        &$\omega_B R_B$
+      \end{tabular}}
+      %
     \item \textbf{Compute} $\omega_B$ and $\alpha_B$ when $t = \frac{\pi}{6}$
       seconds.
-  \end{itemize}
-\end{minipage}
-\hfill
-\begin{minipage}[t]{0.35\linewidth}
-  \minipageTopAnchor
-  \includegraphics[width=1.0\linewidth]{mayflower.jpg}
-  \\[1.5pc]
-  \includegraphics[width=1.0\linewidth]{boat.png}
+      %
+    \\[0.45pc]
+    \Solution{}{0.99\linewidth}{
```

Comment:
Indent the entire solution block another 2 spaces. (I like having the \\[0.45pc] and the \Solution block indented at the same level as the text above it.)